requirements/ingest/sql.in

```diff
@@ -0,0 +1,4 @@
+-c ../constraints.in
+-c ../base.txt
+SQLAlchemy
+psycopg2-binary
```

Comment:
Is the `psycopg2-binary` dependency specific to postgres or all sql connections?

Reply:
then we might want to have this in a `postgres.in` file which extends a `sql.in` file if there are dependencies common across them.